crates/joystick/src/linux.rs

```diff
@@ -1,3 +1,4 @@
+#[cfg_attr(target_os = "linux")]
```

Comment:
Are these needed here since you do a conditional `pub mod` in `lib.rs`?

Reply:
actually a leftover here - good catch.

#[cfg says "only compile this if...."
#[cfg_attr says "if you compile it with this, then use these opts"

in the case of `lib.rs` we want to use the same joystick module call, but we want to redirect the compiler to a specific file depending on operating system.